tools/jil/util/browser-matcher.js

```diff
@@ -238,6 +238,9 @@ features.addEventListenerOptions = new BrowserMatcher()
   .exclude('ios')
   .exclude('ie')
 
+features.supportsFirstInteraction = features.addEventListener
```

Comment:
Do we need this?

Reply:
Nope, I'll remove it. I think originally intended to be `firstInteractionDelay` matcher instead, but ended up unnecessary.. for now.